android/src/main/java/com/reactlibrary/utils/UnsafeConnectionBuilder.java

```diff
@@ -0,0 +1,128 @@
+package com.reactlibrary.utils;
```

Comment:
This file is copied as-is from the AppAuth Android example application. Only modification here is that I changed the name from `ConnectionBuilderForTesting` to `UnsafeConnectionBuilder` https://github.com/openid/AppAuth-Android/blob/29329dce430edebdc80e9635895358e20971f286/app/java/net/openid/appauthdemo/ConnectionBuilderForTesting.java